model/infinispan/src/main/java/org/keycloak/models/cache/infinispan/RealmAdapter.java

```diff
@@ -862,20 +862,16 @@ public void setSmtpConfig(Map<String, String> smtpConfig) {
 
     @Override
     public Stream<IdentityProviderModel> getIdentityProvidersStream() {
```

Comment:
If I understand correctly, the next iteration will be reviewing the places where we are calling this method as it is the root cause of the scalability issues we have today. So we'll be using specific methods in other places to filter brokers using different filters.